### DISCUSSION TITLE
Manually Update

### DISCUSSION THREAD

**Discussion by webmasterra** [Leantime/leantime#1175] [General]
Good day,

I use leantime in my server. (Version:2.2.8) and would like to update manually. How can I update manually (FTP)?

Best regards...

**Comment by marcelfolaron** [comment]
Hi,

the update procedure is quite simple. Just download the latest version and
upload the files to your server (via ftp, replacing existing files).
Configuration files and userdata files will not be impacted.

If there are any database changes the system will redirect you
automatically and execute the db updates.

Sincerely,
Marcel

**Reply by webmasterra** [reply]
Thank you Marcel ) many thanks for the answer. I just put all the server files in via ftp but I get 500 Internal Server Error.

**Reply by webmasterra** [reply]
<img width="270" alt="Bildschirmfoto 2022-12-08 um 08 21 50" src="https://user-images.githubusercontent.com/119939383/206384209-77afbf2c-ca48-447f-9bc0-3e0c30db69e1.png">

should I really upload everything to ftp? or certain folders are not allowed to upload?

**Reply by webmasterra** [reply]
I have now noticed that the data have disappeared from the database. I have backups though.

**Comment by marcelfolaron** [comment]
Any errors in resources/logs/error.log?
Are you still able to login?

**Reply by webmasterra** [reply]
i played my backups again and use the old version.
